realm/realm-library/src/androidTestObjectServer/java/io/realm/SyncUserTests.java

```diff
@@ -80,6 +82,11 @@ public void setUp() {
         SyncManager.reset();
     }
 
+    @After
+    public void after() {
+        UserFactory.logoutAllUsers();
```

Comment:
in this file, there are tests running in the looper thread. Logging out here will make those flaky.